docs/book/src/02_getting_started.md

```diff
@@ -14,7 +14,7 @@ If you don’t already have it installed, you can install Trunk by running
 cargo install trunk
 ```
 
-Create a basic Rust binary project
+Create a basic Rust project
```

Comment:
I'm not sure the word `binary` is helping folks who want to build Leptos apps and binary is the default these days.